docs/team/russelltankaimin.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: Tan Kai Min, Russell's Project Portfolio Page
+---
+
+### Project: EduMate
+
+EduMate is a desktop address book application used to help NUS students maintain both their social and academic life by lowering the barriers to meet up and also make new friends within modules and school. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
```

Comment:
We need to standardise this eventually.

Reply:
I used the one from user guide.

Reply:
I did the same, but what do we want it to be in the end?